specs/src/specs/data_structures.md

```diff
@@ -172,24 +167,18 @@ Unless otherwise indicated explicitly, objects are first [serialized](#serializa
 
 Consensus-critical data is authenticated using [ECDSA](https://www.secg.org/sec1-v2.pdf), with the curve [secp256k1](https://en.bitcoin.it/wiki/Secp256k1). A highly-optimized library is available in C (<https://github.com/bitcoin-core/secp256k1>), with wrappers in Go (<https://pkg.go.dev/github.com/ethereum/go-ethereum/crypto/secp256k1>) and Rust (<https://docs.rs/crate/secp256k1>).
 
-[Public keys](https://docs.cosmos.network/v0.46/basics/accounts.html#public-keys) are encoded in uncompressed form, as the concatenation of the `x` and `y` values. No prefix is needed to distinguish between encoding schemes as this is the only encoding supported.
+[Public keys](https://docs.cosmos.network/v0.46/basics/accounts.html#public-keys) are serialized in a compressed format.
 
 Deterministic signatures ([RFC-6979](https://tools.ietf.org/rfc/rfc6979.txt)) should be used when signing, but this is not enforced at the protocol level as it cannot be.
 
-[Signatures](#signature) are represented as the `r` and `s` (each 32 bytes), and `v` (1-bit) values of the signature. `r` and `s` take on their usual meaning (see: [SEC 1, 4.1.3 Signing Operation](https://www.secg.org/sec1-v2.pdf)), while `v` is used for recovering the public key from a signature more quickly (see: [SEC 1, 4.1.6 Public Key Recovery Operation](https://www.secg.org/sec1-v2.pdf)). Only low-`s` values in signatures are valid (i.e. `s <= secp256k1.n//2`); `s` can be replaced with `-s mod secp256k1.n` during the signing process if it is high. Given this, the first bit of `s` will always be `0`, and can be used to store the 1-bit `v` value.
-
-`v` represents the parity of the `Y` component of the point, `0` for even and `1` for odd. The `X` component of the point is assumed to always be low, since [the possibility of it being high is negligible](https://bitcoin.stackexchange.com/a/38909).
-
-Putting it all together, the encoding for signatures is:
+[Signatures](#signature) are represented as the `r` and `s` (each 32 bytes) values of the signature. `r` and `s` take on their usual meaning (see: [SEC 1, 4.1.3 Signing Operation](https://www.secg.org/sec1-v2.pdf)). The encoding for signatures is:
```

Comment:
@adlerjohn I'd appreciate your review on this section

Reply:
Are signatures _encoded_ as described here, or encoded with protobuf? Also, there's no `v` here?

Reply:
I don't think `v` is supposed to be here. Ref: https://github.com/celestiaorg/celestia-app/issues/1440#issuecomment-1486806399

Reply:
> Are signatures encoded as described here, or encoded with protobuf?

Good point. They appear to be encoded using protobuf per

https://github.com/celestiaorg/cosmos-sdk/blob/f19e20459b8bcf195167ad256de5a6ca6edc41f5/x/auth/tx/encoder.go#L13-L29

Reply:
In that case I would distinguish between the _value_ and the _encoding_ of that value.

Reply:
Agreed. I attempted to in https://github.com/celestiaorg/celestia-app/pull/1553/commits/0d691af1d3c42427dc73ef936b7e055ecfec0524 and https://github.com/celestiaorg/celestia-app/pull/1553/commits/cb52073e9cd3f9f69ef432d9d408db97c7767046